src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -73,33 +73,76 @@
  * </li>
  * </ul>
  * <p>
+ * To run the check with its default configuration (no matches will be):
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;RegexpMultiline&quot;/&gt;
+ * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * void method() {
+ *   int i = 5; // OK
+ *   System.out.println(i); // OK
+ * }
+ * </pre>
+ * <p>
  * To configure the check to find calls to print to the console:
  * </p>
  * <pre>
  * &lt;module name="RegexpMultiline"&gt;
- *   &lt;property name="format"
- *     value="System\.(out)|(err)\.print(ln)?\("/&gt;
+ *   &lt;property name="format" value="System\.(out)|(err)\.print(ln)?\("/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * void method() {
+ *   System.out.print("Example"); // violation
+ *   System.err.println("Example"); // violation
+ *   System.out.print
+ *     ("Example"); // violation
```

Comment:
please do me favor and vertically align trailing comment, not not mix with code visually.